### PR TITLE
Prepare azure-json and azure-xml for July 2024 release

### PR DESCRIPTION
**Pull request by alzimmermsft**

### CODE REVIEW

File: eng/versioning/version_client.txt [hunk 1]
```diff
@@ -200,7 +200,7 @@ com.azure:azure-storage-queue;12.21.1;12.22.0
 com.azure:azure-template-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-template-stress;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-verticals-agrifood-farming;1.0.0-beta.3;1.0.0-beta.4
-com.azure:azure-xml;1.0.0;1.1.0-beta.1
+com.azure:azure-xml;1.0.0;1.1.1
```
Comment (srnagar) [inline eng/versioning/version_client.txt:203]: Let's make this a minor version update as there is contract change in the XmlSerializable interface.

**Review by srnagar — APPROVED** [review]

File: eng/versioning/version_client.txt (hunk 1 above)
Comment (alzimmermsft) [reply eng/versioning/version_client.txt]: Done

**Review by srnagar — APPROVED** [review]

**Review by mssfang — APPROVED** [review]

**Pull request merged**